pkg/version/version.go

```diff
@@ -31,7 +31,7 @@ var (
 	State       string
 
 	// MaxPluginVersion must be increased to add new plugin and make sure the new plugin version is same to this value
-	MaxPluginVersion = 1
+	MaxPluginVersion = 3
```

Comment:
I'm going to have to look through the Migration code in more detail, but if this is actually a version and not something else, then this should be 2 at the most given we've only got one major version released.